packages/botonic-react/src/components/button.jsx

```diff
@@ -137,6 +138,22 @@ export const Button = props => {
           COLORS.SOLID_BLACK
         )
 
+    const urlIconProp = getThemeProperty(WEBCHAT.CUSTOM_PROPERTIES.urlIcon)
+
+    const urlIconEnabled = getThemeProperty(
+      WEBCHAT.CUSTOM_PROPERTIES.urlIconEnabled
+    )
+
+    const urlIconImage = getThemeProperty(
+      WEBCHAT.CUSTOM_PROPERTIES.urlIconImage
+    )
+
+    const urlIcon = urlIconEnabled
+      ? urlIconImage ?? WEBCHAT.DEFAULTS.URL_ICON
+      : propertyDisabled(urlIconEnabled)
+        ? undefined
+        : urlIconImage
+
```

Comment:
You are not using the urlIconProp anywhere. I think this could be simpler, the getThemeProperty function accepts a second parameter which is used in case the first one is undefined.
```suggestion
    const urlIconEnabled = getThemeProperty(
      WEBCHAT.CUSTOM_PROPERTIES.urlIconEnabled,
      false
    )

    const urlCustomIconImage = getThemeProperty(
      WEBCHAT.CUSTOM_PROPERTIES.urlIconImage,
      WEBCHAT.DEFAULTS.URL_ICON
    )

    const urlIcon =
      urlIconEnabled || urlCustomIconImage ? urlCustomIconImage : undefined
```

Reply:
With your solution, the condition will always be true and the url image will always be shown. That's why I'm not using the default value in getThemeProperty function (the second parameter).

Reply:
You are right the condition always returns true and it should not be like that. Keep in mind that maybe you should have a const with the urlCustomIcon and another with the urlDefaultIcon.
Something like the following
```
const urlIconEnabled = getThemeProperty(
      WEBCHAT.CUSTOM_PROPERTIES.urlIconEnabled,
      false
    )
    const urlCustomIconImage = getThemeProperty(
      WEBCHAT.CUSTOM_PROPERTIES.urlIconImage
    )
    const urlDefaulIconImage = WEBCHAT.DEFAULTS.URL_ICON
    const Icon = urlCustomIconImage ? urlCustomIconImage : urlDefaulIconImage
    const urlIcon =
      urlIconEnabled || urlCustomIconImage ? Icon : undefined
```